query-tuning/index.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: 'Query Tuning'
+backlink_href: /docs
+backlink_title: 'Documentation'
+---
+
+import DocsBetaFeature from '../components/DocsBetaFeature'
+
+<DocsBetaFeature feature="Query Tuning" />
+
+## Getting Started
+
+Query Tuning provides a unified interface and workflow for optimizing slow Postgres queries.
+
+The Query Tuning Workbooks feature lets you streamline recording of related `EXPLAIN ANALYZE`
+outputs and enables easy, secure sharing them with your team. Each workbook contains a
+baseline query and one or more variants.
+
+Variants are a modification to the original query that aims to optimize it by changing planner
+settings, adding a `pg_hint_plan` hint, or rewriting the query. You can compare and analyze
+variants with clear visibility into changes and their effects on performance.
+
+The parameter sets feature makes it easy to test performance for different bind parameter values
+across the baseline and variants, making sure that optimizing for one user's experience does not
+slow down other cases.
```

Comment:
I think it may not be obvious to everyone that parameters can affect performance, so we may want to lead with that:
```suggestion
Different parameter values (or different constant values) can lead to varying performance or even a different plan for a specific query.
The parameter sets feature makes it easy to test performance for different bind parameter values
across the baseline and variants, making sure that optimizing for one user's experience does not
slow down other cases.
```